docs/root/intro/arch_overview/http/http3.rst

```diff
@@ -18,42 +18,63 @@ HTTP3 downstream
 Downstream Envoy HTTP/3 support can be turned up via adding
 :ref:`quic_options <envoy_v3_api_field_config.listener.v3.UdpListenerConfig.quic_options>`,
 ensuring the downstream transport socket is a QuicDownstreamTransport, and setting the codec
-to HTTP/3. Please note that hot restart is not gracefully handled for HTTP/3 yet.
+to HTTP/3.
 
-See example :repo:`downstream HTTP/3 configuration </configs/envoyproxy_io_proxy_http3_downstream.yaml>` for example configuration.
+.. note::
+   Hot restart is not gracefully handled for HTTP/3 yet.
 
-Note that the example configuration includes both a TCP and a UDP listener, and the TCP
-listener is advertising http/3 support via an ``alt-svc header``. Advertising HTTP/3 is not necessary for
-in-house deployments where HTTP/3 is explicitly configured, but is needed for internet facing deployments
-where TCP is the default, and clients such as Chrome will only attempt HTTP/3 if it is explicitly advertised.
+.. tip::
 
-By default the example configuration uses kernel UDP support, but for production performance use of
-BPF is strongly advised if Envoy is running with multiple worker threads. Envoy will attempt to
-use BPF on Linux by default if multiple worker threads are configured, but may require root, or at least
-sudo-with-permissions (e.g. sudo setcap cap_bpf+ep). If multiple worker threads are configured, Envoy will
-log a warning on start-up if BPF is unsupported on the platform, or is attempted and fails.
+   See :repo:`downstream HTTP/3 configuration </configs/envoyproxy_io_proxy_http3_downstream.yaml>`
+   for example configuration.
+
+   This example configuration includes both a TCP and a UDP listener, and the TCP
+   listener is advertising HTTP/3 support via an ``alt-svc`` header.
+
+   By default the example configuration uses kernel UDP support, but for production performance use of
+   BPF is strongly advised if Envoy is running with multiple worker threads.
+
+Advertising HTTP/3 is not necessary for in-house deployments where HTTP/3 is explicitly configured,
+but is needed for internet facing deployments where TCP is the default, and clients such as Chrome
+will only attempt HTTP/3 if it is explicitly advertised.
+
+Envoy will attempt to use BPF on Linux by default if multiple worker threads are configured,
```

Comment:
We've got paragraphs on
BPF
advertisement
BPF
BPF

Let's move it around to be contiguous

Reply:
not sure if i follow this feedback exactly, but i think i get the gist

i have added some sub-sections to the http/3 downstream section to give better delineation, and then linked from the config tip where it mentions BPF to the relevant section